doc/tutorial/statistical_inference/model_selection.rst

```diff
@@ -70,7 +70,7 @@ This example shows an example usage of the ``split`` method.
 
 The cross-validation can then be performed easily::
 
-    >>> kfold = KFold(n_splits=3)
+    >>> k_fold = KFold(n_splits=3)
```

Comment:
That's also fair game. Since the snippets below the one referenced for `cross_val_score` don't create another `KFold` instance.